libbeat/docs/shared-ilm.asciidoc

```diff
@@ -0,0 +1,151 @@
+[[ilm]]
+[role="xpack"]
+== Set up index lifecycle management
+
+beta[]
+
+// REVIEWERS: should we mention that this requires a basic license? I started to
+// add a note and then realized that most users will already have a basic license
+// so the note might be confusing.
+
+You can use the {ref}/getting-started-index-lifecycle-management.html[index
+lifecycle management] feature in {es} to manage your {beatname_uc} indices as
+they age. For example, instead of having {beatname_uc} create daily indices
+where index size can vary based on the number of Beats and number of events
+sent, you can use an index lifecycle policy that automates a rollover to a new
+index when the existing index reaches a specified size or age. 
+
+{beatname_uc} provides a default policy that you can load when you set up
+{beatname_uc}. The default policy is applied to any new indices created by
+{beatname_uc}. You can edit the policy to modify the lifecycle of both new and
+existing indices.
+
+To use index lifecycle management on {beatname_uc} indices:
+
+. Enable index lifecycle management by setting `ilm.enabled: true` in the {es}
+output configuration. For example:
++
+--
+[source,yaml]
+------------------------------------------------------------------------------
+output.elasticsearch:
+  hosts: ["localhost:9200"]
+  ilm.enabled: true
+------------------------------------------------------------------------------
+
+This configuration overwrites your index settings and adjusts the {beatname_uc}
+template to use the correct settings for index lifecycle management.
+
+NOTE: If you've previously loaded the index template for this version into {es}, 
+you must overwrite the template by setting `setup.template.overwrite: true`.
+
+The rollover alias is set to +{beatname_lc}-\{beat.version\}+ by default. You
+can change the prefix used in the alias by setting `ilm.rollover_alias`, but you
+can't remove `{beat.version}` from the rollover alias name. The default pattern
+used for the rollover index is `%7Bnow%2Fd%7D-000001`. You can change the
```

Comment:
I actually managed to fix this in one of the last PR's and it's now `"{now/d}-000001"` in the config. Also applies below.